ddtrace/settings/asm.py

```diff
@@ -70,6 +71,7 @@ class ASMConfig(Env):
     )
     _iast_request_sampling = Env.var(float, IAST.ENV_REQUEST_SAMPLING, default=30.0)
     _iast_debug = Env.var(bool, IAST.ENV_DEBUG, default=False, private=True)
+    _iast_context = "_iast_env"
```

Comment:
why have you removed `_IAST_CONTEXT: Literal["_iast_env"] = "_iast_env"`? You can just move it to `appsec/constants` or move as a value of `ddtrace.appsec._constants.IAST`